src/pybind/mgr/cephadm/templates/services/nfs/ganesha.conf.j2

```diff
@@ -31,4 +31,9 @@ RADOS_URLS {
         watch_url = "{{ url }}";
 }
 
+RGW {
+        cluster = "ceph";
+        name = "client.{{ rgw_user }}";
+}
+
```

Comment:
Instead of adding RGW block by default. It would be good to have export type option in `ceph orch apply nfs` or let higher level interfaces modify the ganesha conf according to export type. And we have cluster type option in `volumes/nfs` interface `ceph nfs cluster create <type> <clusterid> [<placement>]` for the purpose of modifying minimal ganesha conf according to export type. @batrick @jtlayton

Reply:
generally agree our goal is to minimize ganesha.conf into a very simple bootstrap config, but we do need to manage the keyrings (both add/remove) and also mount the keyring into each daemon container.

Reply:
I think the problem is that once defined here you can't add other options to the RGW section from another RADOS config. cephadm can setup the keys but I think it'd be better for the higher level config to write the RGW section.

Reply:
I've been able to overwrite this block from the higher level config, but the entire block is replaced, which is problematic because this would require the higher level workflow to also have knowledge of the keyring entity.